### ISSUE TITLE
Write a MarketAdapter for Gemini

### ISSUE THREAD

**Issue by adamrothman** [elsehow/moneybot#63]
This should help us refine the base `MarketAdapter` interface and will in general be an instructive exercise.

https://docs.gemini.com/rest-api/

**Comment by elsehow** [comment]
One more thing about this and #64, which may merit its own issue: Do we want to be able to come up with relationships between markets? For example, maybe we can deposit BTC from Poloniex into Gemini, so as to exchange it for USD.

**Comment by adamrothman** [comment]
Definitely. I think that takes us into the territory of something like a meta-market adapter – a thing that lets us compose markets and trade assets across them.
